src/store/reducers/filter.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export type FilterType = {
+  gender: string[]
+  activeGender: string
+  status: string[]
+  activeStatus: string
+}
+
+export const initialState = {
+  status: ['All', 'Alive', 'Dead', 'Unknown'],
```

Comment:
Tohle by v reduxu appky být nemuselo, je to statický výčet hodnot - v /api/types.ts byl připravený enum.